taglib/src/main/java/com/reevoo/taglib/AbstractReevooMarkClientTag.java

```diff
@@ -119,7 +107,20 @@ protected void setClient(ReevooMarkClient client) {
             // with the parameter "reviews" instead of the parameter "per_page".
             queryStringParams.put("reviews",this.numberOfReviews);
         }
+        if (this.dynamicAttrs != null) {
```

Comment:
dynamicAttrs is an initialized object in AbstractReevooTag, I guess this check could be removed or replaced for checking that it contains something, because right now it's always going to pass.